helix-view-aggregator/src/test/java/org/apache/helix/view/integration/TestHelixViewAggregator.java

```diff
@@ -247,11 +274,16 @@ private void resetViewClusterConfig(int refreshPeriod, List<PropertyType> proper
     _configAccessor.setClusterConfig(viewClusterName, viewClusterConfig);
   }
 
-  private void verifyViewClusterEventChanges(boolean externalViewChange,
-      boolean instanceConfigChange, boolean liveInstancesChange) {
-    Assert.assertEquals(_monitor.getExternalViewChangeCount() > 0, externalViewChange);
-    Assert.assertEquals(_monitor.getInstanceConfigChangeCount() > 0, instanceConfigChange);
-    Assert.assertEquals(_monitor.getLiveInstanceChangeCount() > 0, liveInstancesChange);
+  private Predicate<MockViewClusterSpectator> hasExternalViewChanges() {
```

Comment:
nit : nice use of predicate! may be we can add this method in MockViewClusterSpectator itself!